spec/std/sysctl_spec.cr

```diff
@@ -0,0 +1,54 @@
+require "spec"
+require "sysctl"
+
+describe Sysctl do
+  describe "get_int" do
+    it "returns the sysctl value as an int" do
+      keys = {% if flag?(:darwin) %}
+        ["hw.ncpu", "hw.byteorder"]
+      {% elsif flag?(:linux) %}
+        ["net.ipv4.ip_forward", "kernel.pty.max"]
+      {% else %}
+        []
```

Comment:
Should be `[] of String` (see [failed travis build](https://travis-ci.org/crystal-lang/crystal/jobs/227197195#L168)).